plugins/history.rst

```diff
@@ -1,4 +1,17 @@
 History Plugin
 ==============
 
-TODO: explain the error plugin
+History plugin use a `Http\Client\Plugin\Journal` to collect successful or failed calls of an Http Client. This is
```

Comment:
The HistoryPlugin notifies a `Http\Client\Plugin\Journal` of all successful and failed calls.